docs/outlook/mobile-log-appointments.md

```diff
@@ -514,20 +515,20 @@ Then call it when you want to clear the custom property. For example, you can ca
 ## Test and validate
 
 1. Follow the usual guidance to [test and validate your add-in](testing-and-tips.md).
-1. After you [sideload](sideload-outlook-add-ins-for-testing.md) the add-in in Outlook on the web, Windows, or Mac, restart Outlook on your Android mobile device.
-1. Open an appointment as an attendee then verify that under the **Meeting Insights** card, there's a new card with your add-in's name alongside the **Log** button.
+1. After you [sideload](sideload-outlook-add-ins-for-testing.md) the add-in in Outlook on the web, Windows, or Mac, restart Outlook on your Android or iOS mobile device.
+1. Open an appointment as an attendee, then verify that under the **Meeting Insights** card, there's a new card with your add-in's name alongside the **Log** button.
 
 ### UI: Log the appointment notes
 
 As a meeting attendee, you should see a screen similar to the following image when you open a meeting.
 
-![Screenshot showing the Log button on an appointment screen on Android.](../images/outlook-android-log-appointment-details.jpg)
+![The Log button on an appointment screen on Android.](../images/outlook-android-log-appointment-details.jpg)
```

Comment:
Is the iOS experience significantly different from the Android version? If so, consider including iOS screenshots.

Reply:
Thanks for the feedback. The UX/UI isn't significantly different in iOS, so the current screenshots should suffice. However, I'll add a backlog work item for this as a future update.